.github/workflows/deploy-jupyter-book.yml

```diff
@@ -35,6 +35,11 @@ jobs:
       run: |
         python -m pip list
 
+    - name: Set track_progress=False for notebooks
+      run: |
+        git config --global --add safe.directory /__w/pyhf-tutorial/pyhf-tutorial
+        git grep --name-only "track_progress=True" book | xargs sed --in-place 's/track_progress=True/track_progress=False/g'
```

Comment:
Note that this doesn't do anything for the `Makefile` so that might not be ideal.

Reply:
I asked about this in https://github.com/orgs/executablebooks/discussions/1067.